src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
```

Comment:
this and other implementations should be also placed under 'strategy.impl' package